template-parts/header/site-branding.php

```diff
@@ -18,20 +18,16 @@
 		<div class="site-logo"><?php the_custom_logo(); ?></div>
 	<?php endif; ?>
 
-	<?php if ( ! empty( $blog_info ) && get_theme_mod( 'display_title_and_tagline', true ) === true ) : ?>
-		<?php if ( is_home() ) : ?>
-			<?php if ( is_paged() ) : ?>
-				<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
-			<?php else : ?>
-				<h1 class="site-title"><?php bloginfo( 'name' ); ?></h1>
-			<?php endif; ?>
-		<?php elseif ( is_front_page() ) : ?>
-			<p class="site-title"><?php bloginfo( 'name' ); ?></p>
+	<?php if ( $blog_info && get_theme_mod( 'display_title_and_tagline', true ) === true ) : ?>
```

Comment:
This should use a Yoda syntax, see [WP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions)

```suggestion
	<?php if ( $blog_info && true === get_theme_mod( 'display_title_and_tagline', true ) ) : ?>
```